vxfreeswitch/voice.py

```diff
@@ -50,6 +51,15 @@ def __init__(self, vumi_transport):
         self.input_type = None
         self.uniquecallid = None
 
+    # NOTE: These were lifted straight from eventsocket's github code
+    #       Once https://github.com/fiorix/eventsocket/issues/11 is closed
+    #       This code (and the vumi.log import) can be undone
+    def unknownContentType(self, content_type, ctx):
+        log.debug("[eventsocket] unknown Content-Type: %s" % content_type)
+
+    def unboundEvent(self, ctx, evname):
+        log.debug("[eventsocket] unbound Event: %s" % evname)
```

Comment:
Yes, we can do a `self.vumi_transport.log` here instead of importing `log` from vumi. That will make these messages go through the new Twisted things (which Junebug can catch), instead of the old twisted things.